testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -584,6 +614,12 @@ public Builder setCoordinatorProperties(Map<String, String> coordinatorPropertie
             return this;
         }
 
+        public Builder setBackupCoordinatorProperties(Map<String, String> backupCoordinatorProperties)
+        {
+            this.backupCoordinatorProperties = Optional.ofNullable(backupCoordinatorProperties);
```

Comment:
Use `Optional.of` we do not expect null to be passed here

Reply:
ok

Reply:
> Use `Optional.of` we do not expect null to be passed here

ok